### DISCUSSION TITLE
Remove Videos

### DISCUSSION THREAD

**Discussion by daskandalis** [talmolab/sleap#1363] [Ideas]
Currently videos have to be removed one at a time. Videos can be added in a batch, it would be helpful to have a function to remove them in a batch as well.

**Comment by roomrys** [comment]
Hi @daskandalis,

Thanks for the idea. We've converted this into an issue #1375 and added an implementation in the develop branch #1382 thanks to @gitttt-1234. The video selection requires the user to either click and drag the cursor (to select multiple sequential videos) or to hold `Ctrl` while double clicking on each video rows (to select non-sequential videos).

Thanks,
Liezl

**Comment by roomrys** [comment]
Completed in #1382!
